src/main/java/ch/njol/skript/expressions/ExprLiddable.java

```diff
@@ -0,0 +1,92 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.block.BlockState;
+import org.bukkit.block.Block;
+import org.bukkit.block.Lidded;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Name("Lid State")
+@Description("Set the lid state of a chest, ender chest, barrel and shulker box.")
+@Examples({
+	"if the lid state of {_chest} is false:",
+	"\tset the lid state of {_chest} to true"
+})
+@Since("INSERT VERSION")
+public class ExprLiddable extends PropertyExpression<Block, Boolean> {
+
+	static {
+		Skript.registerExpression(ExprLiddable.class, Boolean.class, ExpressionType.PROPERTY, "[the] lid [state] of %blocks%");
```

Comment:
Needs to allow for `block's lid state` too. Why not just register the expression using PropertyExpression's register method? And this seems like it could be a SimplePropertyExpression tbh